items.js

```diff
@@ -1,28 +1,43 @@
-var items = [
+// items or word that are going to be translated by the user.
+//item: display translation 
+//article: the article translate in learning language( e.g la or le)
+//word: the noun in learning language( needs to fill in), awnser to question
+//translation: the total translation in the learning language
+//audio: the audio reference that pronounces the word(item) in the learning language
+//Used by script.js
+// Carmen Burghardt 29-05-2019
+//Adding some longer words
+
+const items = [
     {
        item: 'the fruit',
```

Comment:
the single item is represented by one single object of this array. I suggest to not call an attribute of this object `item`